utils/stand_monitoring/poller/config.json

```diff
@@ -1,69 +1,29 @@
 {
   "general": {
     "polling_interval": 120,
-    "notification": "slack"
+    "request_timeout": 10,
+    "notification": "slack",
+    "response_pattern": 200
```

Comment:
There is no need in parameterizing respose code, its stable for all http protocol and can be hardcoded.